src/planet_coords.jl

```diff
@@ -54,7 +54,7 @@ Find the RA, Dec of Venus on 1992 Dec 20
 
 ```jldoctest
 julia> adstring(planet_coords(DateTime(1992,12,20),2))
-" 02 12 22.7  +12 44 45"
+" 21 00 15.3  -19 09 09"
```

Comment:
Ok, this has probably to do with the inaccurate method to solve the Kepler's equation in `helio.pro`: I compared the mean anomalies in `helio.pro` and `helio.jl` for a few different inputs and they are exactly equal.